### PR TITLE
Add decisions page route & its integration

### LINKED ISSUE #3859: Redirect `Go back` from decisions details to decisions page instead of home page

### PR DESCRIPTION
**Pull request by chinins**
## Description

This PR separates decisions details route to have a back button pointing to decisions page instead of the home page.

resolves #3859

### CODE REVIEW

**Review by danbr — APPROVED** [review]
Working well - nice fix.

**Review by arrenv — CHANGES_REQUESTED** [review]
Has this been created off the wrong branch?

**Comment by chinins** [comment]
> Has this been created off the wrong branch?

@arrenv, no, it was just not rebased after stuff was merged into the feature branch. It's rebased now - you can review.

File: src/routes/Routes.tsx [hunk 1]
```diff
@@ -235,6 +236,16 @@ const Routes = () => {
             backRoute: `/colony/${colonyName}`,
           })}
         />
+        <AlwaysAccesibleRoute
+          exact
+          path={DECISIONS_PAGE_ROUTE}
+          component={ActionsPage}
+          layout={NavBar}
+          routeProps={({ colonyName }) => ({
+            backText: '',
```
Comment (willm30) [inline src/routes/Routes.tsx:245]: Text here should be "Go back" in order to be consistent with the Preview screen.

**Review by willm30 — APPROVED** [review]
Working nicely, just one observation.

File: src/routes/Routes.tsx (hunk 1 above)
Comment (chinins) [reply src/routes/Routes.tsx:245]: This is consistent with Actions page and works without the extra text
Screenshot 2022-09-20 at 18 31 24
Screenshot 2022-09-20 at 18 31 47

**Review by arrenv — APPROVED** [review]
Works well for me!

**Pull request merged**